web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/dto/behandling/FagsakBehandlingDtoTjeneste.java

```diff
@@ -127,9 +127,9 @@ private FagsakBehandlingDto lagBehandlingDto(Behandling behandling,
             dto.leggTil(get(FamiliehendelseRestTjeneste.FAMILIEHENDELSE_V2_PATH, "familiehendelse-v2", uuidDto));
         }
 
+        kontrollDtoTjeneste.lagKontrollresultatForBehandling(BehandlingReferanse.fra(behandling)).ifPresent(dto::setKontrollResultat);
```

Comment:
Får du ikke nå mange kall til FpRisk for å hente for seneste førstegang?
Er det ikke like greit å populere dto på behandlingnivå kun for førstegang, og så fiske ut en til FagsakNivå i FagsakFullTjeneste.
Da kan frontend defaulte til fagsakens Risiko hvis Dto i enkeltbehandling er null

Reply:
Har lagt til dto'en på FagsakFullDto